provisioning/plugins/corim-psa-decoder/decoder_test.go

```diff
@@ -84,3 +85,47 @@ func TestDecoder_Decode_OK(t *testing.T) {
 		assert.NoError(t, err)
 	}
 }
+
+func TestDecoder_Decode_negative_tests(t *testing.T) {
+	tvs := []struct {
+		desc        string
+		input       string
+		expectedErr string
+	}{
+		{
+			desc:        "multiple verification keys for an instance",
+			input:       unsignedCorimComidPsaMultIak,
+			expectedErr: `bad key in CoMID at index 0: expecting exactly one IAK public key`,
+		},
+		{
+			desc:        "multiple digest for a PSA key in the measurement",
```

Comment:
"PSA key" is a bit ambiguous, maybe:

```suggestion
			desc:        "multiple digests in the same measurement",
```